gui-apps/gtkgreet/gtkgreet-0.6.ebuild

```diff
@@ -37,7 +37,7 @@ BDEPEND="
 src_configure() {
 	local emesonargs=(
 		$(meson_feature man man-pages)
-		$(meson_use layershell)
+		$(meson_use layershell layershell)
```

Comment:
Don't need the second `layershell`.

Reply:
Sure, I did it for consistent look initially.

Reply:
Fixed.